WcaOnRails/app/jobs/update_currency_rates.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class UpdateCurrencyRates < SingletonApplicationJob
+  queue_as :default
+
+  def perform
+    if !Money.default_bank.rates_updated_at || Money.default_bank.rates_updated_at < 1.days.ago
```

Comment:
Extract this `1.days.ago` into a constant along the lines of `CURRENCY_RATES_UPDATE_INTERVAL` maybe? Or is that too overkill?

Reply:
sure, that would be easier for others to edit